File: StoreService/Models/PhoneEntities/PhoneSpec.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace StoreService.Models.PhoneEntities
{
    public class PhoneSpec
    {
        [Key]
        public int Id { get; set; }
        [ForeignKey("Model")]
        public int ModelId { get; set; }
        public PhoneModel Model { get; set; }
        public int StorageGb { get; set; }
        public int RamGb { get; set; }
        public float DisplayIn { get; set; }
        public float CameraMp { get; set; }

    }

}

File: ManagStoreClient/src/app/models/phone/phoneSpec.ts
export interface PhoneSpec {
    id: number;
    storageGb: number;
    ramGb: number;
    displayIn: number;
    cameraMp: number
}
export const phoneSpecs: PhoneSpec[] = [
    {
      id: 1,
      storageGb: 64,
      ramGb: 4,
      displayIn: 6.0,
      cameraMp: 12
    },
    {
      id: 2,
      storageGb: 128,
      ramGb: 6,
      displayIn: 6.4,
      cameraMp: 48
    },
    {
      id: 3,
      storageGb: 256,
      ramGb: 8,
      displayIn: 6.7,
      cameraMp: 108
    }
]